opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -366,8 +370,21 @@ namespace Opm {
             ebosSimulator_.model().linearizer().linearize();
             ebosSimulator_.problem().endIteration();
 
+            // -------- Current time step length ----------
+            const double dt = timer.currentStepLength();
```

Comment:
Please move these two lines to the original place, before Well equations?, since it is closer to where `dt` is used.